cfme/tests/containers/test_containers_default_views.py

```diff
@@ -43,9 +50,11 @@ def test_containers_providers_default_view(button_group, view):
               to Grid/Tile/List view
             * Goes to Compute --> Containers --> Providers and verifies the selected view
         """
-    sel.force_navigate('my_settings_default_views')
+    navigate_to(Server, 'MySettings')
```

Comment:
I'm opening a new PR to provide this destination via navmazing, and to provide a class method for setting default view. If you want, you can close this PR, as I'll include these changes? Or wait until my PR is merged and refactor (this all goes down to like 3 lines)